sklearn/feature_extraction/text.py

```diff
@@ -961,12 +962,17 @@ def _count_vocab(self, raw_documents, fixed_vocab):
                                  " contain stop words")
 
         if indptr[-1] > 2147483648:  # = 2**31 - 1
-            if sp_version >= (0, 14):
+            if sp_version >= (0, 14) and not _IS_32BIT:
```

Comment:
Our current minimal supported scipy version is 0.17, so ou can just remove the scipy version check and remove the last else clause.

Reply:
cool, will update now!